Global/JetBrains.gitignore

```diff
@@ -21,7 +21,7 @@
 
 # Gradle:
 # .idea/gradle.xml
-# .idea/libraries
+# .idea/libraries/Gradle__*.xml
```

Comment:
I gather we're talking about this rule:

> XML files under **.idea/libraries** in case they are **generated from Gradle** project

Just wanted to confirm this is the most suitable expression here (me not being familiar with Gradle and how it's generating these files).